build/bundle.Dockerfile

```
@@ -0,0 +1,20 @@
+FROM scratch
+
+# Core bundle labels.
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=devworkspace-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=alpha
```

```suggestion
LABEL operators.operatorframework.io.bundle.channels.v1=fast
```

as you agreed in the thread https://github.com/devfile/devworkspace-operator/pull/438#discussion_r645555176